dspace-api/src/main/java/org/dspace/storage/bitstore/S3BitStoreService.java

```diff
@@ -258,20 +266,7 @@ public InputStream get(Bitstream bitstream) throws IOException {
         if (isRegisteredBitstream(key)) {
             key = key.substring(REGISTERED_FLAG.length());
         }
-        try {
-            File tempFile = File.createTempFile("s3-disk-copy-" + UUID.randomUUID(), "temp");
-            tempFile.deleteOnExit();
-
-            GetObjectRequest getObjectRequest = new GetObjectRequest(bucketName, key);
-
-            Download download = tm.download(getObjectRequest, tempFile);
-            download.waitForCompletion();
-
-            return new DeleteOnCloseFileInputStream(tempFile);
```

Comment:
Since you've replaced the `DeleteOnCloseFileInputStream`, should we *delete* that class? It looks like it is no longer used.

Reply:
it is still used by the inner class, see https://github.com/DSpace/DSpace/pull/9477/files#diff-02dc8d6ccf99e2499bb006b204a26eb968f83223394b5b50a386d7be887ba8adR682